drivers/storage/partmgr/partmgr.c

```diff
@@ -1206,21 +1208,22 @@ PartMgrAddDevice(
     deviceExtension->IsFDO = TRUE;
     deviceExtension->DeviceObject = deviceObject;
     deviceExtension->LowerDevice = IoAttachDeviceToDeviceStack(deviceObject, PhysicalDeviceObject);
-    deviceExtension->PhysicalDiskDO = PhysicalDeviceObject;
-    KeInitializeEvent(&deviceExtension->SyncEvent, SynchronizationEvent, TRUE);
-
-    // the the attaching failed
     if (!deviceExtension->LowerDevice)
     {
+        /* The attachment failed */
         IoDeleteDevice(deviceObject);
-
         return STATUS_DEVICE_REMOVED;
     }
+    deviceExtension->PhysicalDiskDO = PhysicalDeviceObject;
+    KeInitializeEvent(&deviceExtension->SyncEvent, SynchronizationEvent, TRUE);
```

Comment:
These two code lines were moved in the formatting-only commit, should probably belong to the fix commit.